Intervalos/src/LM/intervalos.jl

```diff
@@ -0,0 +1,321 @@
+        return Intervalo(infimo,supremo)
+    else
+        in_a=getfield(a, :infimo)
+        su_a=getfield(a, :supremo)
+        
+        if isnan(prevfloat(in_a*in_b)) && isnan(prevfloat(su_a*su_b))
+            infimo=0.0
+            supremo=0.0
+        elseif isnan(prevfloat(in_a*in_b))
+            infimo=min(0,prevfloat(in_a*su_b),prevfloat(su_a*in_b),prevfloat(su_a*su_b))
+            supremo=max(0,nextfloat(in_a*su_b),nextfloat(su_a*in_b),nextfloat(su_a*su_b))
+        elseif isnan(prevfloat(in_a*su_b))
+            infimo=min(prevfloat(in_a*in_b),0,prevfloat(su_a*in_b),prevfloat(su_a*su_b))
+            supremo=max(nextfloat(in_a*in_b),0,nextfloat(su_a*in_b),nextfloat(su_a*su_b))
+        elseif isnan(prevfloat(su_a*in_b))
+            infimo=min(prevfloat(in_a*in_b),prevfloat(in_a*su_b),0,prevfloat(su_a*su_b))
+            supremo=max(nextfloat(in_a*in_b),nextfloat(in_a*su_b),0,nextfloat(su_a*su_b))
+        elseif isnan(prevfloat(su_a*su_b))
+            infimo=min(prevfloat(in_a*in_b),prevfloat(in_a*su_b),prevfloat(su_a*in_b),0)
+            supremo=max(nextfloat(in_a*in_b),nextfloat(in_a*su_b),nextfloat(su_a*in_b),0)
+        else 
+            infimo=min(prevfloat(in_a*in_b),prevfloat(in_a*su_b),prevfloat(su_a*in_b),prevfloat(su_a*su_b))
+            supremo=max(nextfloat(in_a*in_b),nextfloat(in_a*su_b),nextfloat(su_a*in_b),nextfloat(su_a*su_b))
+        end
+        return Intervalo(infimo,supremo)
+    end
+end
+
+*(a::Intervalo,b::Real)=*(b,a)
+
+import Base: /
+
+function /(a::Real,b::Intervalo)
+    if 0 ∈ b 
+        if b==Intevalo(0.0)
+            return intervalo_vacio(typeof(in_b))
+        elseif in_b < 0 < b_su
+            return Intervalo(-Inf, Inf)
+        elseif 0== in_b < su_b
+            return Intervalo(prevfloat(in_a/su_b),Inf)
+        elseif in_b < su_b ==0
+            return Intervalo(-Inf,nextfloat(in_a/in_b))
+        end
+    end
+
+    infimo=min(prevfloat(a/in_b),prevfloat(a/su_b))
+    supremo=max(nextfloat(a/in_b),nextfloat(a/su_b))
+    return Intervalo(infimo,supremo)
+end
+
+function /(a::Intervalo,b::Intervalo)
+    in_a=getfield(a, :infimo)
+    su_a=getfield(a, :supremo)
+    in_b=getfield(b, :infimo)
+    su_b=getfield(b, :supremo)
+
+    if isnan(in_a)
+        return a
+    elseif isnan(in_b)
+        return b
+    end
+
+    if 0 ∈ b
+        return division_extendida(a,b)
+    end
+
+    infimo=min(prevfloat(in_a/in_b),prevfloat(in_a/su_b),prevfloat(su_a/in_b),prevfloat(su_a/su_b))
+    supremo=max(nextfloat(in_a/in_b),nextfloat(in_a/su_b),nextfloat(su_a/in_b),nextfloat(su_a/su_b))
+    return Intervalo(infimo,supremo)
+end
+
+function /(a::Intervalo,b::Real)
+    if b==0
+        return error("El denominador es cero")
+    end
+    return a*Intervalo(1/b)
+end
+
+
+# Definimos la función para las potencias enteras
+
+import Base: ^
+
+function ^(a::Intervalo,b::Int64)
+    if b==0
+        return Intervalo(1,1)
+    end
+    
+    if isnan(getfield(a, :infimo)) || b=1
```

Comment:
```suggestion
    if isnan(getfield(a, :infimo)) || b==1
```
Recuerda que en las comparaciones se deben usar dos iguales ya que uno indica una asignación.